.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "god.tsconfig": "./tsconfig.json"
```

Comment:
is this config for `TypeScript God`? maybe the repo should include a list of "suggested" addons because apparently i'm missing out on god mode

Reply:
yeaha i actually removed this plugin from VSCode, since it seems to hhandle TS just ok on its own. i'll kill thjis